python-toolbox/marvin_python_toolbox/management/notebook.py

```diff
@@ -29,15 +29,15 @@ def cli():
 
 @cli.command('notebook', help='Start the Jupyter notebook server.')
 @click.option('--port', '-p', default=8888, help='Jupyter server port')
+@click.option('--base-url', '-p', default='/', help='Jupyter server base url')
```

Comment:
The short parameter for base url could be something else than '-p' that conflicts with Port´s short parameter.